app/tests/compilers/css.js

```diff
@@ -0,0 +1,4 @@
+function noop() { return null; }
+
+require.extensions['.css'] = noop;
```

Comment:
shoudl there also be an extension for .jsx?

current webpack config allows these to be imported without an extension, but they won't be picked up by tests. Also does this work for imports without extension?